### PR TITLE
Mkirk 2.19.3/fixup account cache

### PR DESCRIPTION
**Pull request by michaelkirk**
PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/contact/OWSContactsManager.m [hunk 1]
```diff
@@ -273,6 +273,10 @@ - (void)updateSignalAccounts:(NSArray<SignalAccount *> *)signalAccounts
     self.signalAccountMap = [signalAccountMap copy];
     self.signalAccounts = [signalAccounts copy];
     [self.profileManager setContactRecipientIds:signalAccountMap.allKeys];
+
```
Comment (michaelkirk) [inline Signal/src/contact/OWSContactsManager.m]: This is critical

File: Signal/src/ViewControllers/NewContactThreadViewController.m [hunk 2]
```diff
@@ -396,14 +396,34 @@ - (void)updateTableContents
...
+                    addItem:[OWSTableItem softCenterLabelItemWithText:
+                                              NSLocalizedString(@"SETTINGS_BLOCK_LIST_NO_CONTACTS",
+                                                  @"A label that indicates the user has no Signal contacts.")
+                                                      customRowHeight:self.actionCellHeight]];
+            } else {
+                UITableViewCell *loadingCell = [UITableViewCell new];
+                OWSAssert(loadingCell.contentView);
+
+                UIActivityIndicatorView *activityIndiactorView =
+                    [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/NewContactThreadViewController.m]: This is nice to have, and most users will rarely see it.

File: Signal/src/ViewControllers/NewContactThreadViewController.m [hunk 3]
```diff
@@ -396,14 +396,34 @@ - (void)updateTableContents
...
+                [contactsSection
+                    addItem:[OWSTableItem softCenterLabelItemWithText:
+                                              NSLocalizedString(@"SETTINGS_BLOCK_LIST_NO_CONTACTS",
+                                                  @"A label that indicates the user has no Signal contacts.")
+                                                      customRowHeight:self.actionCellHeight]];
+            } else {
+                UITableViewCell *loadingCell = [UITableViewCell new];
+                OWSAssert(loadingCell.contentView);
+
+                UIActivityIndicatorView *activityIndiactorView =
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/NewContactThreadViewController.m]: sp: indicator

**Comment by michaelkirk** [comment]
in 162b33ed (targeting hotfix, not master, than RI'd)

**Pull request closed without merging**